products/oraclelinux.md

```diff
@@ -17,29 +18,33 @@ auto:
 releases:
 -   releaseCycle: "9"
     releaseDate: 2022-07-06
-    support: 2032-07-01
-    eol: 2034-06-01
+    eol: 2032-07-01
+    extendedSupport: 2034-07-01
```

Comment:
According to https://www.oracle.com/a/ocom/docs/elsp-lifetime-069338.pdf, GA, EOL and extended support dates are all on June, not July. This is consistent with https://blogs.oracle.com/linux/post/announcing-oracle-linux-9-general-availability, which announce the GA on 2022-30-06.

Nothing can be done for `releaseDate` (auto = distrowatch), we should update both EOL and extended support dates.

Reply:
Fixed in 655fbff29f3403b6dd35b6e55b2c77264fb08f7b.